contracts/index_executor/src/lib.rs

```diff
@@ -74,10 +65,10 @@ mod index_executor {
     #[derive(Clone, Encode, Decode, Debug)]
     #[cfg_attr(feature = "std", derive(scale_info::TypeInfo, StorageLayout))]
     pub struct Config {
-        /// The rollup anchor pallet id on the target blockchain
-        rollup_pallet_id: u8,
-        /// The endpoint of rollup deployed chain
-        rollup_endpoint: String,
+        /// The storage provider url
+        storage_url: String,
```

Comment:
Maybe you can just call it `database_url`. Storage is misleading because it's hard to tell if it's a local storage, an on-chain storage, or something else.

Reply:
Renamed